README.md

```diff
@@ -100,7 +129,11 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema lays out the structure of the tables within the database, and sets the
+data type for each column (name would be a VARCHAR type, age would be INTEGER).
+A one-to-many relationship is represented by adding the foreign key of the "one"
+as an attribute on the "many". So, each wish would have a "person_id" attribute
+that would refer back to the person.
```

Comment:
:+1:  Couldn't have said it better myself.  There has to be a "The needs of the many outweigh the needs of the one." pun in there somewhere.  There has to be.